### DISCUSSION TITLE
Select All Filters / Select None Filters

### DISCUSSION THREAD

**Discussion by bandaid** [uBlockOrigin/uBlock-issues#2029] [Ideas] [answered]
We need a Select All Filters / Select None Filters in filter dashboard lists. __

**Comment by u-RraaLL** [comment, marked as answer]
Already declined. https://github.com/uBlockOrigin/uBlock-issues/issues/172#issuecomment-413836348

**Reply by bandaid** [reply]
> Already declined. [#172 (comment)](https://github.com/uBlockOrigin/uBlock-issues/issues/172#issuecomment-413836348)

Gorhill is overreacting.

**Reply by ghost** [reply]
Select All Filters is ridiculous, just will weaken filtering and increase breakages.

**Reply by bandaid** [reply]
it doesn't. been using my 60+ filters for over a decade. not a single problem or "breakage"

**Reply by u-RraaLL** [reply]
Good for you.

Now please understand that your lucky streak means nothing to the team and volunteers who actually have to deal with invalid issues from people using non-default filter lists.

**Reply by bandaid** [reply]
> Good for you.
> 
> Now please understand that your lucky streak means nothing to the team and volunteers who actually have to deal with invalid issues from people using non-default filter lists.

Good for you and the typical pathetic attitude in this generation. 

"lucky streak" ... lol. I modify my filters when needed and post discrepancies.
